### PR TITLE
Setting up ADO PR build

### PR DESCRIPTION
**Pull request by pradeepananth**
- Setting up ADO PR build for live share package. Runs build step for Live share packages and samples.
- To fix SDL step, modified innerHTML to key attribute.
- clean up of samples, remove redundant `if` block for initialized state.

### CODE REVIEW

File: packages/live-share-inking/src/core/LiveCanvas.ts [hunk 1]
```diff
@@ -259,7 +259,7 @@ class BuiltInLiveCursor extends LiveCursor {
         }
 
         const template = document.createElement("template");
-        template.innerHTML = visualTemplate;
+        template["innerHTML"] = visualTemplate;
```
Comment (dclaux) [inline packages/live-share-inking/src/core/LiveCanvas.ts:262]: Just curious why this change is necessary?

Comment (pradeepananth) [reply packages/live-share-inking/src/core/LiveCanvas.ts:262]: Enabling Teams Build requires Guardian's team's SDL pipeline running and that enforces security rules like "no-inner-html" as modifying HTML DOM directly is a security violation.

image

File: packages/updateDependencies.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+ // 
+ //     node updateDependencies.js
+ // 
+ // PUBLISHING:
+ //
+ //     node updateDependencies.js 1.0.0
+ //
+ //=================================================================================================
+ 
+ const localDependencies = {
```
Comment (Stevenic) [inline packages/updateDependencies.js:24]: We should eventually update this to update the sample version numbers as well. It's ok to skip for now.

File: .azure-devops/live-share-ci.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+- template: variables/variables.yml
+- name: esLintExclusionFilePath
+  value: "$(Build.SourcesDirectory)/.eslintignore"
+- name: esLintTargets
+  value: ''
+
+parameters:
+- name: version
+  type: string
+  default: "0.1.0"
```
Comment (Stevenic) [inline .azure-devops/live-share-ci.yml:22]: >"0.1.0" [](http://example.com/codeflow?start=11&length=7)

how does this get set?

**Review by Stevenic — APPROVED** [review]
:shipit:

File: .azure-devops/live-share-ci.yml (hunk 3 above)
Comment (pradeepananth) [reply .azure-devops/live-share-ci.yml:22]: when running ADO pipeline

image

**Comment by Stevenic** [comment]
I left some comments/questions but generally looks ok to me

File: packages/updateDependencies.js (hunk 2 above)
Comment (pradeepananth) [reply packages/updateDependencies.js:24]: sure, this initial check in is mostly for PR build, will have to figure out the CI/package publish automation story.
